src/sentry/services/hybrid_cloud/app/serial.py

```diff
@@ -43,6 +45,7 @@ def serialize_sentry_app_installation(
 ) -> RpcSentryAppInstallation:
     if app is None:
         app = installation.sentry_app
+        assert app  # narrow types
```

Comment:
I'd remove the comment and put `assert app is not None` since that's clear it's a type narrowing